### ISSUE TITLE
macro parameters are not highlighted

### ISSUE THREAD

**Issue by jeff-hykin** [jeff-hykin/cpp-textmate-grammar#327]
```cpp
#define HARVEST_REQUEST_PARAM_APPEND(string, key, value, edited, ampersand)                        \
	do {                                                                                           \
		if (edited) {                                                                              \
			if (ampersand) {                                                                       \
				g_string_append_printf(string, "&%s=%s", #key, #value);                            \
			} else {                                                                               \
				ampersand = TRUE;                                                                  \
				g_string_append_printf(string, "%s=%s", #key, #value);                             \
			}                                                                                      \
		}                                                                                          \
	} while (0)
```
the #key and #value

**Comment by jeff-hykin** [comment]
> Wouldn't adding macro_argument to ever_present_context recreate #292?

I was never able to reproduce #292 and we could always add a lookaround to prevent it from matching preprocessor directives

**Comment by jeff-hykin** [comment]
Fixed with `1.13.0`
